### DISCUSSION TITLE
Docker Image

### DISCUSSION THREAD

**Discussion by ArrichM** [sgl-project/sglang#217] [Show and tell]
Thank you so much for the great work you have done with sglang, it has so far been a really great experience for our use cases.

I created a Docker image to run an sglang server with flashinfer installed. The image is built on top of the official [vllm image](https://docs.vllm.ai/en/latest/serving/deploying_with_docker.html) . To run it, use:

docker run --runtime nvidia --gpus all -v ~/.cache/huggingface:/root/.cache/huggingface -p 8000:8000 --ipc=host arrichm/sglang:latest --model-path mistralai/Mistral-7B-v0.1 --host 0.0.0.0

Here:
--model-path: specifies the model you want to serve
--host 0.0.0.0: Makes the endpoint accessible from outside the container.

If not already installed, you need to install the [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html) before running.

**Comment by tomheno** [comment]
Hello @ArrichM 
I know this is a bit outdate, but would it be possible for you to provide the used DockerFile ?

Thank in advance !

**Comment by merrymercy** [comment]
Done https://github.com/sgl-project/sglang?tab=readme-ov-file#method-3-using-docker
